modules/F50_base_aggregator.sh

```diff
@@ -319,20 +342,21 @@ output_binaries() {
           DATA=1
         fi  
         (( i++ ))
-      done
-    fi  
-  fi 
-  if [[ $DATA -eq 1 ]]; then
-    print_bar
+        done
+      fi  
+    fi 
+    if [[ $DATA -eq 1 ]]; then
+      print_bar
+    fi
   fi
-
 }
 
 output_cve_exploits() {
 
   local DATA=0
   if [[ "$S30_VUL_COUNTER" -gt 0 || "$CVE_COUNTER" -gt 0 || "$EXPLOIT_COUNTER" -gt 0 ]]; then
     print_output "[*] Identified the following software inventory, vulnerabilities and exploits:"
+    write_link "f19"
```

Comment:
could we also link to anchors like: file:///home/m1k3/git-repos/emba_logs/logs_dir300/html-report//f19_cve_aggregator.html#collectcveandexploitdetails